backend/src/controllers/course.js

```diff
@@ -59,7 +59,7 @@ exports.getCourseReviews = function ({ params, query }, res) {
                 'meta': {
```

Comment:
why are these quotes? seems like extra hassle.

Reply:
not in pr :'( @Aruzal